src/styles/main.scss

```
@@ -1,3 +1,59 @@
-body {
+@use "sass:list";
+@import './variables';
+@import './reset';
+
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  box-sizing: border-box;
```

you can just use, bc `flex-direction: row` by default
```suggestion
  flex-wrap: wrap;
```

---

Would be better move  `box-sizing: border-box;` to `*` selector

---

flex-flow: row wrap; - linter recommended me use  flex-flow for obtain flex-direction & flex-wrap

---

src/styles/calendar.scss
 4:3  ✖  Expected shorthand property "flex-flow"  declaration-block-no-redundant-longhand-properties